### ISSUE TITLE
Reporting some typos in the documentation

### ISSUE THREAD

**Issue by pkawak** [NVIDIA/nvmath-python#33]
Thank you for the helpful docs at https://docs.nvidia.com/cuda/nvmath-python/latest/index.html

I'm not sure if this is the appropriate place to report these but after going through the docs, I found some typos that I would like to report.

1. At the [Overview](https://docs.nvidia.com/cuda/nvmath-python/latest/overview.html) page, under "Full Logging Support," the code block is indented incorrectly.
2. At [Implementing a neural network using nvmath-python](https://docs.nvidia.com/cuda/nvmath-python/latest/tutorials/notebooks/matmul/03_backpropagation.html), the loss function under the "Network Architecture" section is missing its second term (but it is correct in the binary_cross_entropy function).
3. At [Implementing a neural network using nvmath-python](https://docs.nvidia.com/cuda/nvmath-python/latest/tutorials/notebooks/matmul/03_backpropagation.html), the "Gradients" section has a sentence edited in math mode.
4. I'm not sure if this is a Python version thing but in copying over the codes, I had to change the cmap arguments from Grays to Greys for the code to work ([Implementing a neural network using nvmath-python](https://docs.nvidia.com/cuda/nvmath-python/latest/tutorials/notebooks/matmul/03_backpropagation.html) page). According to the matplotlib docs, it is Greys and not Grays.
